about/email_sign_up.md

```diff
@@ -0,0 +1,3 @@
+<div id="muNBrVxMCA">
```

Comment:
```suggestion
Email Updates
======

<div id="muNBrVxMCA">
```

Reply:
Above suggestion will look like this:

<img width="1163" alt="Screen Shot 2021-01-29 at 1 00 32 PM" src="https://user-images.githubusercontent.com/746253/106310905-4b87a600-6232-11eb-81e4-883f2be430b9.png">